### PR TITLE
APD-194 - rename settings to use UNIVERSAL_NOTIFICATIONS namespace

### PR DESCRIPTION
**Pull request by poxip**
Renamed:

```
IS_SECURE -> UNIVERSAL_NOTIFICATIONS_IS_SECURE
UNSUBSCRIBE_MODEL -> UNIVERSAL_NOTIFICATIONS_UNSUBSCRIBE_MODEL
DEFAULT_FROM_EMAIL -> UNIVERSAL_NOTIFICATIONS_DEFAULT_FROM_EMAIL
SEND_SMS_FUNC -> UNIVERSAL_NOTIFICATIONS_SEND_SMS_FUNC
TESTING -> UNIVERSAL_NOTIFICATIONS_TESTING
FCM_API_KEY -> UNIVERSAL_NOTIFICATIONS_FCM_API_KEY
GCM_API_KEY -> UNIVERSAL_NOTIFICATIONS_GCM_API_KEY
GCM_POST_URL -> UNIVERSAL_NOTIFICATIONS_GCM_POST_URL
MOBILE_APPS -> UNIVERSAL_NOTIFICATIONS_MOBILE_APPS
```

### CODE REVIEW

File: universal_notifications/backends/push/fcm.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 def fcm_send_message(device, message, data=None):
     app_settings = get_app_settings(device.app_id)
-    api_key = app_settings.get('FCM_API_KEY')
+    api_key = app_settings.get('UNIVERSAL_NOTIFICATIONS_FCM_API_KEY')
```
Comment (remik) [inline universal_notifications/backends/push/fcm.py]: This is in `UNIVERSAL_NOTIFICATIONS_MOBILE_APPS` so no need to prefix again.

File: universal_notifications/backends/emails.py [hunk 2]
```diff
@@ -40,6 +40,6 @@ def send_email(template, to, subject, variables={}, fail_silently=False, cms=Fal
                      include_star_selectors=True,
                      strip_important=False,
                      base_url=base).transform()
-    email = EmailMessage(subject, html, settings.DEFAULT_FROM_EMAIL, [to])
+    email = EmailMessage(subject, html, settings.UNIVERSAL_NOTIFICATIONS_DEFAULT_FROM_EMAIL, [to])
```
Comment (remik) [inline universal_notifications/backends/emails.py]: This is django variable so you should check both.

File: universal_notifications/backends/push/gcm.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def _chunks(l, n):
 
 def _gcm_send(app_id, data, content_type):
     app_settings = get_app_settings(app_id)
-    key = app_settings.get('GCM_API_KEY')
+    key = app_settings.get('UNIVERSAL_NOTIFICATIONS_GCM_API_KEY')
```
Comment (remik) [inline universal_notifications/backends/push/gcm.py]: Again no need prefix.

File: universal_notifications/backends/push/gcm.py [hunk 4]
```diff
@@ -45,7 +45,7 @@ def _gcm_send(app_id, data, content_type):
         "Content-Length": str(len(data)),
     }
 
-    request = Request(SETTINGS["GCM_POST_URL"], data, headers)
+    request = Request(SETTINGS["UNIVERSAL_NOTIFICATIONS_GCM_POST_URL"], data, headers)
```
Comment (remik) [inline universal_notifications/backends/push/gcm.py]: This is external library variable

File: universal_notifications/backends/websockets.py [hunk 5]
```diff
@@ -17,7 +17,7 @@ def publish(user, item=None, additional_data=None):
     data.update(additional_data)
     data = r.render(data)
     message = RedisMessage(data)
-    if settings.TESTING:
+    if settings.UNIVERSAL_NOTIFICATIONS_TESTING:
```
Comment (remik) [inline universal_notifications/backends/websockets.py]: Please add checking if it is test.

File: universal_notifications/backends/emails.py (hunk 2 above)
Comment (pkrzyzaniak) [inline universal_notifications/backends/emails.py]: it django variable so it is pointless to rename it - reverting

**Pull request merged**